sympy/physics/mechanics/_actuator.py

```diff
@@ -0,0 +1,243 @@
+"""Implementations of actuators for linked force and torque application.
+
+Notes
+=====
+
+This module is experimental and so is named with a leading underscore to
+indicate that the API is not yet stabilized and could be subject to breaking
+changes.
+
+"""
+
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+
+from sympy.core.backend import Basic
+from sympy.core.expr import Expr
+from sympy.physics.mechanics import Point, Vector
+from sympy.physics.mechanics._pathway import PathwayBase
+
+
+__all__ = ['ForceActuator']
+
+
+class ActuatorBase(ABC):
+    """Abstract base class for all actuator classes to inherit from.
+
+    Notes
+    =====
+
+    Instances of this class cannot be directly instantiated by users. However,
+    it can be used to created custom actuator types through subclassing.
+
+    """
+
+    def __init__(self) -> None:
```

Comment:
It's there purely as a placeholder. I wanted the concrete classes to call `super().__init__()` in their own initialisers so that if the ABC is changed then it doesn't break anything unexpectedly. I also wanted it to be clear what was being called in that `super().__init__()` call, hence the blank initialiser in the ABC. Yes, it's currently superfluous, but I don't think it's harmful. Happy to remove both this and the `super().__init__()` call in concrete classes if you request.